CheckerServer/CheckerServer/Models/OrderItem.cs

```diff
@@ -15,11 +15,11 @@ public class OrderItem : BaseDBItem
         public int ServingAreaZone { get; set; }
 
         public string Changes { get; set; }
-        public ItemStatus Status { get; set; } = ItemStatus.Ordered;
-        public LineItemStatus LineStatus { get; set; } = LineItemStatus.Locked;
+        public eItemStatus Status { get; set; } = eItemStatus.Ordered;
+        public eLineItemStatus LineStatus { get; set; } = eLineItemStatus.Locked;
     }
 
-    public enum ItemStatus      // the status of the order item overall
+    public enum eItemStatus      // the status of the order item overall
     {
         Ordered,                // initial state
         AtLine,                 // during entire time it is at preperation line
```

Comment:
In-Progres  instead At-Line

Reply:
AtLine is what was approved previously for clarity, I see no need to change now.